functions/notify_slack.py

```diff
@@ -72,15 +72,19 @@ def notify_slack(payload):
     return result
 
 
-def get_guardduty_event(bucket, key):
+def get_guardduty_events(bucket, key):
     s3_client = boto3.client("s3")
     response = s3_client.get_object(Bucket=bucket, Key=key)
 
     object_bytes = response["Body"].read()
     object_string = gzip.decompress(object_bytes).decode("utf-8")
+    object_lines = object_string.splitlines()
+    
+    guardduty_events = []
```

Comment:
(nit)

```
return [json.loads(line) for line in object_lines]
```

I think might be slightly more pythonic. I guess you could use `map` but I think that fell out of favour due to arguments of being less-clear. Either way, i'm not actually expecting you to change anything.

Reply:
Again, thanks for the suggestion, i've done that, personally it's not just more pythonic, but also removes 4 lines of code that a bug could be hiding in totally think it is worth it.